azerothcore/azerothcore-wotlk
# Fix spellclick

## Changes Proposed:
-  
-  

## Issues Addressed:

- Closes 

## SOURCE:


## Tests Performed:

- Tested
- Arch Linux


## How to Test the Changes:


1. See video without PR
2. https://user-images.githubusercontent.com/14080516/135752124-d2da5222-da8a-4087-b0ae-77b8f5e6d990.mp4
3. Download https://github.com/IntelligentQuantum/WoWAddin
4. Compile
5. Run WoWAddin.exe

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
